MediaBrowser.Model/Configuration/ServerConfiguration.cs

```diff
@@ -165,6 +165,12 @@ public ServerConfiguration()
         /// <value>The file watcher delay.</value>
         public int LibraryMonitorDelay { get; set; } = 60;
 
+        /// <summary>
+        /// Gets or sets the duration in seconds that we will wait after a library updated event before executing the library changed notification.
+        /// </summary>
+        /// <value>The library update duration.</value>
+        public int LibraryUpdateDuration { get; set; } = 30;
```

Comment:
This is in seconds but the `dueTime` parameter expects milliseconds, making the timer now fire every 30ms.